scripts/updateapilist.py

```diff
@@ -292,20 +303,11 @@ def mono_repo_clients(token: str) -> List[CloudClient]:
     # all mono repo clients
     url = f"{BASE_API}/repos/{MONO_REPO}/contents/packages"
     headers = {'Authorization': f'token {token}'}
-    try:
-        response = requests.get(url=url, headers=headers)
-        response.raise_for_status()
-    except requests.RequestException as e:
-        logging.error(f"Request failed for URL {url}: {e}")
+    _, metadata = _fetch_and_parse_response(url, headers)
```

Comment:
Nit nit (not a blocker; you can leave as is)

```suggestion
    _, packages = _fetch_and_parse_response(url, headers)
```